build.gradle.kts

```diff
@@ -79,7 +80,7 @@ dependencies {
     forge("net.minecraftforge:forge:1.8.9-11.15.1.2318-1.8.9")
 
     // Discord RPC client
-    shadowImpl("com.github.NetheriteMiner:DiscordIPC:3106be5") {
+    shadowImpl("com.jagrosh:DiscordIPC:0.5") {
```

Comment:
@NetheriteMiner your feedback please

Reply:
I will note that the version change is because my fork added the button functionality 3 years ago and all other forks have not added them.

Reply:
This was resolved on discord with netheriteminer